lib/cartopy/examples/nightshade.py

```diff
@@ -0,0 +1,23 @@
+"""
+Nightshade feature
+------------------
+
+Shade areas of night terminator
+
+"""
+__tags__ = ['Lines and polygons']
+
+import datetime
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+from cartopy.feature.nightshade import Nightshade
+
+
+ax = plt.axes(projection=ccrs.PlateCarree())
+
+date = datetime.datetime(1999, 12, 31, 12)
+
+plt.title('Night time shading for {}'.format(date))
```

Comment:
`ax.set_title`

Reply:
OK I accept your cleanliness 😜

Reply:
I also *attempted* to improve the description.